definitions/npm/axios_v0.15.x/flow_v0.28.x-/axios_v0.15.x.js

```diff
@@ -93,6 +93,7 @@ declare module 'axios' {
       request: AxiosRequestInterceptor<mixed>,
       response: AxiosResponseInterceptor<mixed>,
     };
+    defaults: AxiosXHRConfig<T>
```

Comment:
Maybe we can try to ask past contributors?

@marudor sorry to bother, I see you worked on previous iterations of the axios interface definition -- if you have a minute do you think you could point us in the right direction? Thank you.